cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+    cy.fixture('discount/discountschema.json').then(discountSchemaJson => {
+      Object.assign(new DiscountSchema(), discountSchemaJson)
+        .setName(discountSchemaName)
+        .apply();
+    });
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for transport unit', function() {
+    // eslint-disable-next-line
+    Builder.createProductWithPriceUsingExistingCategory(priceListName, productForPackingMaterialTU, productForPackingMaterialTU, productType, "24_Gebinde");
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for loading unit', function() {
+    Builder.createProductWithPriceUsingExistingCategory(
+      priceListName,
+      productForPackingMaterialLU,
+      productForPackingMaterialLU,
+      productType,
+      '24_Gebinde'
+    );
+  });
+  it('create packing material for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForTU)
+        .setProduct(productForPackingMaterialTU)
+        .setLength('0')
+        .setWidth('0')
+        .setHeight('0')
+        .apply();
+    });
+  });
+  it('create packing material for handling unit - Load unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForLU)
+        .setProduct(productForPackingMaterialLU)
+        .setLength('0')
+        .setWidth('0')
+        .setHeight('0')
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForTU)
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Load unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForLU)
+        .apply();
+    });
+  });
+  it('create packing instruction version for handling unit - Transport unit; with packgut - quantity=0.00 and packmittel', function() {
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsVersionForTU)
+        .setPackingInstructions(packingInstructionsNameForTU)
+        .setPackingMaterial(packingMaterialForTU)
+        .apply();
+    });
+    /**For some reasons,the quantity for the packgut cannot be added in advanced edit mode, only like below */
```

Comment:
Imo, this should be added to `PackingInstructionsVersion.apply_M_HU_PI_Item()`, since maybe other tests will need to set it as well. (please find a better method name :D)

Reply:
I am not sure I understand your suggestion. Can you please give me more details?

Reply:
You're creating a new Packing Instructions Version via the normal builder. 
But afterwards you are editing the qty in the `M_HU_PI_Item` tab. This qty edit part should be part of the builder js object (and set in the normal `apply()` method in the builder.

Basically you extract this part to that builder instead of keeping in this test, and maybe also copying it to other tests which may need to set the qty to a certain value.

Reply:
The quantity for packgut is not needed anymore. Removed it.

Reply:
I have tried to do the adding of a position to be more generic but I had several errors and there are too many corner cases in this scenario. For example, when adding a Packgut - the quantity can be added only in the table by double clicking in the column cell. If you choose Unter-packvorschrift - the quantity can be added in the **new window** and also in there you can add a Packing instruction(Included_HU_PI_ID). If you choose Packmittel - you can only add a packing material(M_HU_PackingMaterial_ID) in the **new window**. For the time being I'll leave it as it is. If you encounter any other test which will need a similar scenario in the future, hand it over to me or feel free to adapt it now.